<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frameTaskPanel</class>
 <widget class="QDialog" name="frameTaskPanel">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="lOffset">
         <property name="text">
          <string>Offset :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lx">
         <property name="text">
          <string>x :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="sbWidth">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>9999999999999999455752309870428160.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lWidth">
         <property name="text">
          <string>Width :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="sbOffset">
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-100000000000000004188152556421145795899143386664033828314342771180699648.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999999999999999932209486743616279764617084419440640.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lHeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Height :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbHeight">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>99999999999999997748809823456034029568.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="sbx">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ckConstrained">
     <property name="text">
      <string>Constrained dimensions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb8">
     <property name="text">
      <string>8 points</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb12">
     <property name="text">
      <string>12 points</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb16">
     <property name="text">
      <string>16 points</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
